src/test/java/com/google/api/codegen/testdata/discoveries/foo.v1.json.overrides

```diff
@@ -4,5 +4,27 @@
   },
   "js|python": {
     "discoveryDocUrl": "localhost:8080/$discovery/foo?version=v1"
+  },
+  "csharp|go|java": {
+    "methods": {
+      "foo.bar.get": {
+        "fields": {
+          "foobar": {
+            "defaultValue": "\" \""
```

Comment:
Technically, yes, but it could be annoying in the future if we do so. For example, we might want to insert `null`, or `x.toString();` or something like that without quotations.